assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md
+* Compliance guys have to do a job; this is made difficult by teams resisting this.
+* Teams are focusing on business value; CONTRIBUTING.md compliance is viewed as not as important.
+* Special task force for security and compliance: developers responsible for fixing these bugs across the company.
+* You have to do these fixes.
+* Negotiations are being held but they are slow and little progress is made.
+* The task force is discovering the needed information for a CONTRIBUTING.md as they investigate the problem.
+* There is a checklist for being InnerSource compliant.
+* There is a need to be Security compliant.
+* There may be export control compliance and legal compliance requirements; a template is provided to repository owners
+
+## Forces
+* Teams have been resisting this; this ends up wasting time.
+* Making documents part of the repo skeleton might be "rubber stamping"; better to have teams own this. So too much automation in this case is bad.
+
+## Solution
+* Rather than asking the resisting team to do the changes, they create the documentation (in addition to negotiations)
+* Taking the contributor perspective (contributors are motivated). They are writing the CONTRIBUTING.md documentation for those teams resistant to doing the fixes, doing this as pull requests. The discussion is then documented in the pull request. The resisting development teams then just correct mistakes.
+* "Let us help you be compliant"
+* You could do an audit to assess the state of compliance. Bots could be used to check compliance; and the state of compliance could show up in an internal portal.
```

Comment:
I propose to make this less conditional.

Reply:
While it is ideal to persuade rather than to enforce, the fact that this pattern was written shows that persuasion may not be sufficient. I agree that getting people to understand the benefits should be something tried first (and this should be part of the pattern).

Reply:
Thanks for the clarification, @NewMexicoKid.

Reply:
@gregswindle I know that this conversation here is a loooong while back. I just wanted to let you know what we have merged the pattern to the main line by now. 

I am also curious to learn more about your project https://github.com/commonality/generator-community and what experiences you made with it. If you would like to chat about it, feel free to reply here or join the community in https://innersourcecommons-inviter.herokuapp.com/ :)